static/index.html

```diff
@@ -60,25 +61,25 @@ <h1>Conference about Open Source</h1>
             <form>
               <div class="form-wrapper col-12">
                 <input type="text" placeholder="Email address" class="input-text col-7">&emsp;
-                <input type="submit" value="Submit" class="submit-btn col-4">
+                <button type="button" class="btn btn-primary col-4">Submit</button>
               </div>
             </form>
           </div>
         </div>
       </div>
     </header>
-    <section class ="content">
-      <div class = "content-wrapper">
+    <section class="content">
+      <div class="content-wrapper">
         <div class="container">
           <div class="row">
-            <div class = "sponsor-region col-lg-12">
+            <div class="sponsor-region col-lg-12">
               <h2>Sponsors</h2>
               <p>Your contribution will help lower ticket prices for attendees, and as a sponsor, there is something in it for you</p>
-              <div class = "link-btn">  
-                <a href="#">Sponsor now</a>
+              <div class="sponsor-btn">
+                <button type="button" class="btn btn-outline-primary">Sponsor Now</button>
               </div>
               <h4>Gold Sponsors</h4>
-              <div class ="sponsors-image">
+              <div class="sponsors-image">
                 <img src="images/Promet_Logo_Lockup.png"  alt="Promet Source logo.">
```

Comment:
@alvincases please wrap img tag inside <a> tag and input href="https://www.prometsource.com/".
<a href="https://www.prometsource.com/"><img></a>